apps/passport-client/components/screens/ProveScreen/SemaphoreGroupProveScreen.tsx

```diff
@@ -37,17 +36,13 @@ export function SemaphoreGroupProveScreen({
     fetchGroup().catch(console.error);
   }, [req.args.group.remoteUrl]);
 
-  // Once that's done & user clicks Prove, create a zero-knowledge proof
-  const [state] = useContext(DispatchContext);
-  const [proving, setProving] = useState(false);
-
   const onProve = useCallback(async () => {
     try {
       setProving(true);
 
       // Give the UI has a chance to update to the 'loading' state before the
       // potentially blocking proving operation kicks off
-      sleep(200);
+      await sleep(200);
```

Comment:
i think this can be shorter, like `sleep(50)`;

Reply:
might put in `util.js` so u have something like `await nextFrame()`

Reply:
done